lib/Alien/Base/ModuleBuild.pm

```diff
@@ -343,6 +347,10 @@ sub ACTION_alien_code {
     $self->config_data( system_provides => \%system_provides );
     return;
   }
+  
+  if ($ForceSystem) {
+    die "Requested $ENV{ALIEN_INSTALL_TYPE} install, but system package not detected."
```

Comment:
In my experience it is more generic and robust to use perl capabilities
instead of relying on the non-perl environment.

On Fri, Jul 17, 2015 at 2:53 PM, Joel Berger <notifications@github.com>
wrote:

> In lib/Alien/Base/ModuleBuild.pm
> <https://github.com/Perl5-Alien/Alien-Base/pull/135#discussion_r34920067>:
>
> > @@ -343,6 +347,10 @@ sub ACTION_alien_code {
> >      $self->config_data( system_provides => \%system_provides );
> >      return;
> >    }
> > +
> > +  if ($ForceSystem) {
> > +    die "Requested $ENV{ALIEN_INSTALL_TYPE} install, but system package not detected."
>
> is it redundant to interpolate the ENV var here? Seems like system would
> be fine
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Perl5-Alien/Alien-Base/pull/135/files#r34920067>.
>